README.md

```diff
@@ -1,77 +1,65 @@
+2. The system can recover from failures and not exit until the file has been successfully downloaded
+3. The file is deleted if the integrity check fails after downloading. You can use the Content-MD5 for this: https://www.oreilly.com/library/view/http-the-definitive/1565925092/re17.html
+4. Progress is reported to the user throughout the download
+5. The user can cancel the download
 
-There is already a ```WebSystemCalls.java``` and corresponding implementation which allows you to get the HTTP headers for a URL, download the whole content or download the partial content. All these calls return an ```HttpResponse``` object, which contains sub properties for headers and the content.
+### What does a successful submission look like?
+A high-quality submission will have the following:
+- Meet the requirements above
+- Appropriate use of language features to solve the problem in a neat and simple way
+- Code that is easy to read and reason about. Typically using software design principles like SOLID.
+- Unit tests
 
-As in the example here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Ranges, the HTTP header "Accept Ranges" will be set to "Bytes" if the CDN supports partial content.
+_If you feel that modifying the skeleton project would create a better solution to these requirements, we encourage you to do so. We currently support submissions in either .NET (C#) or Java and have included skeleton projects for both._
 
-A test project is included (with JUnit added though can be swapped for an alternative) if Unit tests are wanted. There is also an App Main function which has a real Accurx URL (which **DOES** support partial content) and an example of a file path so that the code can be tested for real.
 
 ## Tips
 
 - Take the time to read through the task and description. There's guidance in there that can be helpful to approaching the problem.
 - The code doesn't need to be perfect but it needs to be readable.
-- Try writing down some example input and outputs on paper.
+- Try writing down some example inputs and outputs on paper.
 - Try a brute force approach and then optimise the code.
 - Add some comments to your code if you think it will be helpful to share your thought process to someone assessing it.
 - You can throttle your internet connection using NetLimiter or similar.
-- You can simulate internet disconnections through disconnecting wifi/ethernet.
+- You can simulate internet disconnections by disconnecting wifi/ethernet.
 - Different behaviours occur after following different periods of disconnection, two seconds and two minutes are sweet spots for exercising key failure modes.
-- Usage of NuGet packages/third party libraries is fine, however do bear in mind that overusing these limits your ability to show off!
\ No newline at end of file
+- The use of NuGet packages/third-party libraries is fine, however, do bear in mind that overusing these limits your ability to show off!
```

Comment:
```suggestion
- The use of third-party libraries is fine, however, do bear in mind that overusing these may limit your ability to demonstrate your skills
```
NuGet is .NET specific, so I'd suggest that we just say third-party libraries unless we want to move this into language specific READMEs.